cert-tool/cert-tool.sh

```diff
@@ -449,6 +457,61 @@ function load_certs() {
 }
 
 
+# DESC: Load current state
+# ARGS: none
+# OUTS: None if successful, Error text otherwise
+function load_current_state() {
+    INGRESS_GENERATION=$(${oc_cmd} get deployment router-default -n openshift-ingress -o json | jq -r '.status.observedGeneration')
```

Comment:
Just a note, wouldn't it be faster to run only one call against ARO API and then parse response with `jq`. Ie:
```bash
ROUTER_RESOURCE=$(${oc_cmd} get deployment router-default -n openshift-ingress)
INGRESS_GENERATION=$(echo ${ROUTER_RESOURCE} | jq -r '.status.observedGeneration')
...
```

Reply:
Done